test/serverTest.js

```diff
@@ -0,0 +1,161 @@
+const sleep = (millis) => new Promise(resolve => setTimeout(resolve, millis))
+const chai = require('chai')
+const chaiHttp = require('chai-http')
+const expect = chai.expect
+const { after, afterEach, beforeEach, describe, it } = require('mocha')
+const imports = require('../index.js')
+const db = imports.db
+const redis = imports.redis
+const server = imports.server
+const XETHRU_STATE = require('../SessionStateXethruEnum.js');
```

Comment:
Can you please remove all the semi-colons in this file?

Reply:
yes!